### ISSUE TITLE
Please Write Detailed Installation Steps

### ISSUE THREAD

**Issue by MiyaNoctem** [michaelmcandrew/pop#6]
Seems like a great tool! Unfortunately, I wasn't able to use it, as I haven't the faintest idea of how to install this in my dev environment. I use buildkit to work with CiviCRM. I'd imagine you somehow have to include the library into cv and generate a new phar? Or am I missing something?

Thanks!

**Comment by michaelmcandrew** [comment]
Hey there,

At the moment, the easiest way to use this is to replace the default `cv` with [this fork of `cv`](https://github.com/michaelmcandrew/cv/tree/pop). No need to generate a new phar. You can just point to https://github.com/michaelmcandrew/cv/blob/pop/bin/cv.

Let me know if you have any other questions.
Michael

**Comment by igorbabo** [comment]
Hi, 
I have same problem as MiyaNoctem, the instructions on installation are really confusing to say at least.
I have replaced default cv with yours fork and it's not working. This is the exception:
[Symfony\Component\Console\Exception\CommandNotFoundException]  
  Command "pop" is not defined.                                   
  Did you mean one of these?                                      
      php:script                                                  
      php:eval                                                    
      php:boot                                                    
 Also, I have tryed installing it with composer and I'we got the same error. 
It would be great if You extended installation guide or give some detailed information on installation. 

Best regards,
Igor
